File: 9-palindrome-number/9-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x < 0 :
            return False
        else:
            b=x
            rev = 0
            while x > 0 :
                rem = x % 10
                rev = rev *10 + rem
                print(rev)
                x = x//10
            if rev != b:
                return False
            else :
                return True

File: 9-palindrome-number/9-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        unsigned int newnum = 0;
        int copy = x ;
        
            
        while(x>0){
            
            newnum = newnum*10 + (x%10) ;
            x = x/10;
            
        }
        if (newnum==copy){
            return true ;
        }
        return false;
        
    }
};